manifests/server/config.pp

```diff
@@ -140,7 +140,13 @@
       mode   => $::puppet::server_environments_mode,
     }
 
-    # make sure your site.pp exists (puppet #15106, foreman #1708)
+    # make sure your site.pp exists (puppet #15106, foreman #1708) and server_manifest_path too
+    file { $puppet::server_manifest_path:
+      ensure => directory,
+      owner  => $puppet::server_user,
+      group  => $puppet::server_group,
+      mode   => '0755',
+    } ->
```

Comment:
The puppet agent errors quoted above were observed trying to run theforeman-puppet module on a CentOS 7 host with python 2.7.  Adding this file resource made the errors go away.
``
# rpm -qa | grep python-2.7
python-2.7.5-18.el7_1.1.x86_64
``
It could be the particular version of puppetserver RPM pulled in neglects to create the path at $server_manifest_path.
```
# rpm -qa | grep puppetserver
puppetserver-2.1.1-1.el7.noarch
```